lib/Basics/StringBuffer.cpp

```diff
@@ -904,11 +895,13 @@ int arangodb::basics::StringBuffer::inflate(arangodb::basics::StringBuffer& out,
       TRI_ASSERT(res != Z_STREAM_ERROR);
 
       switch (res) {
-      case Z_NEED_DICT:
-      case Z_DATA_ERROR:
-      case Z_MEM_ERROR: {
-        return TRI_ERROR_INTERNAL;
-      }
+        case Z_NEED_DICT:
+        case Z_DATA_ERROR:
+        case Z_MEM_ERROR: {
+          return TRI_ERROR_INTERNAL;
+        }
+        default:
```

Comment:
same here, although this looks slighty different (`;` on new line here) than the previous occurrence.